gow.go

```diff
@@ -101,6 +101,7 @@ var (
 
 func main() {
 	FLAG_SET.Usage = func() {}
+	FLAG_SET.StringVar(&CMD, "g", "go", "")
```

Comment:
Super minor quibble: this doesn't need to duplicate the default value. It could refer to itself:
```go
FLAG_SET.StringVar(&CMD, "g", CMD, "")
```
Alternatively, maybe the variable should be defined as a flag to begin with:
```go
FLAG_CMD = FLAG_SET.String("g", "go", "")
```
This is very minor. I could also "fix" this after the merge.